README-zh_CN.md

```diff
@@ -14,6 +14,7 @@
 
 |![更新](static/bell-outline-badge.svg) 最新进展：|
 |------------------|
+|**最新发布**：2023年4月17日. Fluid v1.0.0  发布! 相关功能等版本更新内容请详情参见 [CHANGELOG](CHANGELOG.md)。|
```

Comment:
I think it's 2024.

Reply:
> I think it's 2024.

Thanks for the reviewing. I will fix it.

Reply:
> And how about only keeping the first and latest release info?

OK, that's make it short but keep all the important release news currently.